audbackend/core/backend/base.py

```diff
@@ -781,6 +817,7 @@ def open(
 
         """
         utils.call_function_on_backend(self._open)
```

Comment:
Maybe we check if a connection was already established?

Reply:
I changed to:
```python
        if not self.opened:
            utils.call_function_on_backend(self._open)
            self.opened = True
```